fdbserver/ConsistencyScan.actor.cpp

```diff
@@ -447,16 +456,47 @@ ACTOR Future<Void> checkDataConsistency(Database cx,
 		// If the destStorageServers is non-empty, then this shard is being relocated
 		state bool isRelocating = destStorageServers.size() > 0;
 
+		state int customReplication = configuration.storageTeamSize;
+		if (g_network->isSimulated() && SERVER_KNOBS->DD_MAXIMUM_LARGE_TEAMS > 0 &&
+		    !SERVER_KNOBS->SHARD_ENCODE_LOCATION_METADATA) {
+			for (auto& it : g_simulator->customReplicas) {
+				if (range.begin < std::get<1>(it) && std::get<0>(it) < range.end) {
```

Comment:
As we are trying to see if those two ranges intersect here, is it a better idea to do in this way:
```
KeyRangeRef tmpReplicatedRange(std::get<0>(it), std::get<1>(it));
if (range.intersect(tmpReplicatedShard)) {
...
}
```
I believe they are doing the same thing, just some thoughts here.